CONTRIBUTING.md

```diff
@@ -29,14 +29,14 @@ accurate comments, etc.) and any other requirements (such as test coverage).
 
 4.  To run all packages tests simply do `npm test` (we use [jest](https://github.com/facebook/jest)). If you want to work on a specific package and run the tests only for that package, we recommend to use `npm run test:package`. This will prompt you to select one of the available packages. To run in _watch_ mode simply do `npm run test:package -- --watch`.
 
-5.  Integration tests are separated out in another folder "/integration-tests". To run the integration test, you need to create a env file as specified [here](https://commercetools.github.io/nodejs/sdk/api/getCredentials.html). Then run integration test with
+5.  Integration tests are separated out in another folder "/integration-tests". To run the integration test, you need to export the environment variables into the process env by doing the following :-
 
 ```
-npm run test:integration -- --projectkey=testing-project --runInBand
+    - Export `npm_config_projectkey`=`projectKey`,
+    - Export `CT_PROJECT_KEY`=`CLIENT_ID`:`CLIENT_SECRET` as specified [here](https://commercetools.github.io/nodejs/sdk/api/getCredentials.html).
```

Comment:
This does not work. Please have a look at the preview
https://github.com/commercetools/nodejs/blob/00b8d15f44dbe8a5d6adb25cc07da66e329a1f86/CONTRIBUTING.md